docs/reference/announcements.md

```diff
@@ -132,6 +132,27 @@ Starting with Camunda 8.6, the Camunda Optimize artifact has been split into two
 
 Make sure to update your Docker configurations accordingly to ensure compatibility.
 
+### New base path for Operate and Tasklist Webapps
+
+We are introducing a new base path for both the Operate and Tasklist **web applications**. This change applies to both self-managed and SaaS environments.
+
+#### For Self-Managed
+
+- The new base path for Operate webapp is `/operate`, and for Tasklist webapp, it is `/tasklist`.
+- The full URL for Operate webapp will be `{hostname}/operate` or `{hostname}/{operate-contextPath}/operate`, and similarly for Tasklist.
+- Any calls to `{hostname}` or `{hostname}/{operate-contextPath}` will automatically be redirected to `{hostname}/operate` or `{hostname}/{operate-contextPath}/operate` respectively, with the same logic applying to Tasklist.
```

Comment:
We need to make it clear that the 1st case works only with [Separated Ingress](https://docs.camunda.io/docs/self-managed/setup/guides/ingress-setup/?ingress=separated) (`{hostname}` redirects to `{hostname}/operate` ), and the 2nd case works only with [Combined Ingress](https://docs.camunda.io/docs/self-managed/setup/guides/ingress-setup/?ingress=combined) (`{hostname}/operate` redirects `{hostname}/{operate-contextPath}/operate`)

It's also important to know that the [Separated Ingress is deprecated in 8.6 and will be removed in 8.7](https://docs.camunda.io/docs/next/self-managed/operational-guides/update-guide/850-to-860/#separated-ingress-deprecation)

Reply:
I did that in this commit https://github.com/camunda/camunda-docs/pull/4334/commits/ded1842dbeeed04ad7aad9a06e8a817e25dc623c